images/hook/entrypoint.sh

```diff
@@ -67,17 +70,34 @@ if [ $1 = "update" ]; then
         fi
     fi
 
-    echo "---> Creating new 'grafana' password"
+    # Generate password for Grafana administrator
+    password=$(cat /dev/urandom | tr -dc 'a-zA-Z0-9' | fold -w 32 | head -n 1 | tr -d '\n ' | /opt/bin/base64)
+    sed -i s/cGFzc3dvcmQtZ29lcy1oZXJlCg==/$password/g /var/lib/gravity/resources/secrets.yaml
+
+    # Generate password for InfluxDB administrator
+    password=$(cat /dev/urandom | tr -dc 'a-zA-Z0-9' | fold -w 32 | head -n 1 | tr -d '\n ' | /opt/bin/base64)
+    sed -i s/b1lIV3gyVDlmQVd3SzdsZTRrZDY=/$password/g /var/lib/gravity/resources/secrets.yaml
+
+    # Generate password for InfluxDB grafana user
+    password=$(cat /dev/urandom | tr -dc 'a-zA-Z0-9' | fold -w 32 | head -n 1 | tr -d '\n ')
+    sed -i s/cGRGY29ma2NHYllIekRZMUdadmg=/$(echo -n $password | /opt/bin/base64)/g /var/lib/gravity/resources/secrets.yaml
```

Comment:
Also - why is the grafana user created differently as compared to all other commands?

Reply:
Probably it was original legacy code and for other variables I wrote my own.